test/integration/fixtures/build-withConfig/tsdx.config.js

```diff
@@ -3,6 +3,10 @@ const autoprefixer = require('autoprefixer');
 const cssnano = require('cssnano');
 
 module.exports = {
+  /**
+   * @param {import('tsdx').RollupOptions} config
+   * @param {import('tsdx').TsdxOptions} options
```

Comment:
```suggestion
   * @param {import('../../../../').RollupOptions} config
   * @param {import('../../../../').TsdxOptions} options
```

You had `require('..')` before, so it should either be that (`'..'`) or this, which works in VSCode but may not work during the test phase where this gets moved to the "staging" dir. But `tsdx build` can't type-check this anyway (otherwise we'd be using `tsdx.config.ts`). Might add `tsc` tests in the future for things like #871 though...